### PR TITLE
Adds documentation about combined HA and Authentication uses

### PR DESCRIPTION
**Pull request by IvanVas**
### Description 
Discussion: https://confluentcommunity.slack.com/archives/C6UJNMY67/p1658922529620649
Fixes https://github.com/confluentinc/ksql/issues/8256

### Testing done 
Previewed the docs.

### Reviewer checklist
- [x] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [x] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: docs/operate-and-deploy/high-availability.md [hunk 1]
```diff
@@ -41,6 +41,19 @@ parameter. For instance, a value of `10,000` means that results of pull queries
 forwarded to servers whose current offset is more than `10,000` positions
 behind the end offset of the changelog topic are rejected.
 
+## Compatability with Authentication
+`authentication.skip.paths` should be set with both `/lag` and `/heartbeat`.
```
Comment (JimGalasyn) [inline docs/operate-and-deploy/high-availability.md:45]:
```suggestion
Set the `authentication.skip.paths` config with both `/lag` and `/heartbeat`.
```

File: docs/operate-and-deploy/high-availability.md [hunk 2]
```diff
@@ -41,6 +41,19 @@ parameter. For instance, a value of `10,000` means that results of pull queries
...
+This would allow ksqlDB cluster instances to communicate without authenticating between each other.
```
Comment (JimGalasyn) [inline docs/operate-and-deploy/high-availability.md:46]:
```suggestion
This enables ksqlDB cluster instances to communicate without authenticating between each other.
```

File: docs/operate-and-deploy/high-availability.md [hunk 3]
```diff
@@ -41,6 +41,19 @@ parameter. For instance, a value of `10,000` means that results of pull queries
...
+
+
+The configuration prevents the error below.
```
Comment (JimGalasyn) [inline docs/operate-and-deploy/high-availability.md:49]:
```suggestion
This configuration prevents the following error.
```

File: docs/operate-and-deploy/high-availability.md [hunk 4]
```diff
@@ -41,6 +41,19 @@ parameter. For instance, a value of `10,000` means that results of pull queries
...
+```
+ksqldb-server1     | [...] ERROR Failed to handle request 401 /heartbeat (io.confluent.ksql.api.server.FailureHandler:38)
+ksqldb-server1     | io.confluent.ksql.api.server.KsqlApiException: Unauthorized
+```
+
+It's recommended to block the traffic from outside your cluster to those endpoints for security reasons.
```
Comment (JimGalasyn) [inline docs/operate-and-deploy/high-availability.md:55]:
```suggestion
For security reasons, this configuration is recommended to block the traffic from outside your cluster to those endpoints.
```

File: docs/operate-and-deploy/high-availability-pull-queries.md [hunk 5]
```diff
@@ -79,4 +79,16 @@ ensure that your pull queries succeed with controlled lag and low latency.
 
 !!! note 
     {{ site.ccloud }} is configured with HA enabled by default on clusters 8 CSUs or more.
- 
\ No newline at end of file
+
+## Compatability with Authentication
+`authentication.skip.paths` should be set with both `/lag` and `/heartbeat`.
```
Comment (JimGalasyn) [inline docs/operate-and-deploy/high-availability-pull-queries.md:84]:
```suggestion
Set the `authentication.skip.paths` config with both `/lag` and `/heartbeat`.
```

File: docs/operate-and-deploy/high-availability-pull-queries.md [hunk 6]
```diff
@@ -79,4 +79,16 @@ ensure that your pull queries succeed with controlled lag and low latency.
...
+This would allow ksqlDB cluster instances to communicate without authenticating between each other.
```
Comment (JimGalasyn) [inline docs/operate-and-deploy/high-availability-pull-queries.md:85]:
```suggestion
This enables ksqlDB cluster instances to communicate without authenticating between each other.
```

File: docs/operate-and-deploy/high-availability-pull-queries.md [hunk 7]
```diff
@@ -79,4 +79,16 @@ ensure that your pull queries succeed with controlled lag and low latency.
...
+
+
+The configuration prevents the error below.
```
Comment (JimGalasyn) [inline docs/operate-and-deploy/high-availability-pull-queries.md:88]:
```suggestion
This configuration prevents the following error.
```

File: docs/operate-and-deploy/high-availability-pull-queries.md [hunk 8]
```diff
@@ -79,4 +79,16 @@ ensure that your pull queries succeed with controlled lag and low latency.
...
+```
+ksqldb-server1     | [...] ERROR Failed to handle request 401 /heartbeat (io.confluent.ksql.api.server.FailureHandler:38)
+ksqldb-server1     | io.confluent.ksql.api.server.KsqlApiException: Unauthorized
+```
+
+It's recommended to block the traffic from outside your cluster to those endpoints for security reasons.
```
Comment (JimGalasyn) [inline docs/operate-and-deploy/high-availability-pull-queries.md:94]:
```suggestion
For security reasons, this configuration is recommended to block the traffic from outside your cluster to those endpoints for security reasons.
```

**Review by JimGalasyn — APPROVED** [review]
Thank you for the contribution! LGTM, with a few suggestions.

**Review by agavra — APPROVED** [review]
Thanks @IvanVas - this will hopefully help a lot of people going forward! I'll merge it after addressing Jim's copy edits

**Comment by IvanVas** [comment]
> Thanks @IvanVas - this will hopefully help a lot of people going forward! I'll merge it after addressing Jim's copy edits

Done, thanks!

**Pull request merged**